python/pyarrow/tests/test_pandas.py

```diff
@@ -3374,10 +3371,7 @@ def test_conversion_extensiontype_to_extensionarray(monkeypatch):
     arr = pa.ExtensionArray.from_storage(MyCustomIntegerType(), storage)
     table = pa.table({'a': arr})
 
-    with pytest.raises(ValueError):
-        table.to_pandas()
-
-    # patch pandas Int64Dtype to have the protocol method
+    # ensure pandas Int64Dtype has the protocol method (for older pandas)
```

Comment:
Whould we monkey-patch selectively (only if Pandas is old enough)?

Reply:
> Whould we monkey-patch selectively (only if Pandas is old enough)?

The problem is that the cut-off is somewhere in current master (not yet released). Although, since we only care about *latest* master, that should actually be doable, yes.